source/index.html.md

```diff
@@ -1576,34 +1576,35 @@ GET https://gateway.clearhaus.com/account
 
 ## Transaction status codes
 
-The JSON response will include one of following transaction status codes when
-you make a new transaction.
-
-Status     | `code` |  Meaning
----------- | ------ | --------
-Approved   |  20000 |  Approved
-Declined   |  40000 |  General input error
-           |  40110 |  Invalid card number
-           |  40111 |  Unsupported card scheme
-           |  40120 |  Invalid CSC
-           |  40130 |  Invalid expire date
-           |  40135 |  Card expired
-           |  40140 |  Invalid currency
-           |  40150 |  Invalid text on statement
-           |  40190 |  Invalid transaction
-           |  40200 |  Clearhaus rule violation
-           |  40300 |  3-D Secure problem
-           |  40310 |  3-D Secure authentication failure
-           |  40400 |  Backend problem
-           |  40410 |  Declined by issuer or card scheme
-           |  40411 |  Card restricted
-           |  40412 |  Card lost or stolen
-           |  40413 |  Insufficient funds
-           |  40414 |  Suspected fraud
-           |  40415 |  Amount limit exceeded
-           |  40416 |  Additional authentication required
-           |  40420 |  Merchant blocked by cardholder
-           |  50000 |  Clearhaus error
+When you make a new transaction the JSON response includes one of the status
+codes in the following table. The rightmost columns indicate which transaction
+types a particular status code can be received for; Au, Ca, Re, Vo and Cr are
+short for authorizations, captures, refunds, voids and credits, respectively.
+
+Status     | `code` |  Meaning                            | Au      | Ca      | Re      | Vo      | Cr      |
```

Comment:
I think this will be easier to read if we get rid of the abbreviations and just use the (almost) full words in the table:

```suggestion
types a particular status code can be received for.

<div id="transaction-status-codes-table"></div>
Status     | `code` |  Meaning                            | Auth    | Capture | Refund  | Void    | Credit  |
```

With a little bit of CSS, this will become: 
![Transaction status codes table](https://user-images.githubusercontent.com/1747366/181757202-b3863626-b3ba-4b91-9dac-4011678cd88e.png)


```css
#transaction-status-codes-table + table td:nth-child(n+4) {
  text-align: center;
}

#transaction-status-codes-table + table th:nth-child(n+4) {
  font-variant-caps: all-small-caps;
}
```

Reply:
Thanks a bunch :+1: 

Added in https://github.com/clearhaus/gateway-api-docs/pull/134/commits/ca4861087c78b3115185dca636f355ea7c9d09ad.